trailblazer/store/crud/update.py

```diff
@@ -1,10 +1,42 @@
+from datetime import datetime
+
+from trailblazer.constants import TrailblazerStatus
 from trailblazer.store.base import BaseHandler_2
-from trailblazer.store.models import User
+from trailblazer.store.models import User, Analysis
 
 
 class UpdateHandler(BaseHandler_2):
     """Class for updating items in the database."""
 
+    def add_pending_analysis(
```

Comment:
If we're following CRUD should this not be in a `CreateHandler`?

Reply:
I think you are right. Anything using insert should go into create. I will move it.